pkg/server/authserver/authentication_test.go

```diff
@@ -89,20 +90,33 @@ func TestSSLEnforcement(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 	ctx := context.Background()
-	s := serverutils.StartServerOnly(t, base.TestServerArgs{
+	srv := serverutils.StartServerOnly(t, base.TestServerArgs{
 		// This test is verifying the (unimplemented) authentication of SSL
 		// client certificates over HTTP endpoints. Web session authentication
 		// is disabled in order to avoid the need to authenticate the individual
 		// clients being instantiated.
 		InsecureWebAccess: true,
 	})
-	defer s.Stopper().Stop(ctx)
+	defer srv.Stopper().Stop(ctx)
+
+	if srv.TenantController().StartedDefaultTestTenant() {
+		// Enable access to the nodes endpoint for the test tenant.
+		_, err := srv.SystemLayer().SQLConn(t, "").Exec(
+			`ALTER TENANT [$1] GRANT CAPABILITY can_view_node_info=true`, serverutils.TestTenantID().ToUint64())
+		require.NoError(t, err)
+
+		serverutils.WaitForTenantCapabilities(t, srv, serverutils.TestTenantID(), map[tenantcapabilities.ID]string{
+			tenantcapabilities.CanViewNodeInfo: "true",
+		}, "")
```

Comment:
Not necessary for this PR, but I wonder what you think of integrating a call to `ts.tenantCapabilitiesWatcher.TestingRestart()` into WaitForTenantCapabilities to avoid these tests slowing down too much when we add these wait calls.

Reply:
Ok. Added a commit at the end to do that.